dot/telemetry/telemetry.go

```diff
@@ -23,6 +23,8 @@ const (
 	txPoolImportMsg = "txpool.import"
 )
 
+//go:generate mockgen -destination=mock_telemetry_client.go -package=$GOPACKAGE . Client
```

Comment:
Can you have this as a `_test.go` file to reduce Go API pollution? 🤔

Reply:
Updated.